src/ddmd/errors.d

```diff
@@ -19,6 +19,16 @@ import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.console;
 
+/**********************
+ * Color highlighting to classify messages
+ */
+enum Classification
+{
+    error = Color.red,          // for errors
+    gagged = Color.magenta,     // for gagged errors
+    warning = Color.yellow,     // for warnings
+    _deprecation = Color.blue,  // for deprecations
```

Comment:
Nit: the convention (at least in Phobos) is `keyword_`, i.e. this should be `deprecation_`.